spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/bigquant/Quant.scala

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.nn.bigquant
+
+import com.intel.analytics.bigdl.Module
+import com.intel.analytics.bigdl.nn.bigquant.Utils._
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import java.nio.ByteBuffer
+import scala.reflect.ClassTag
+
+object Quant {
+  def findMax(src: Array[Float], start: Int, end: Int): Float = {
+    src.slice(start, end).max
+  }
+
+  def findMin(src: Array[Float], start: Int, end: Int): Float = {
+    src.slice(start, end).min
+  }
+
+  def quantize(value: Float, max: Float, min: Float): Byte = {
+    Math.round(1.0 * value / Math.max(Math.abs(max), Math.abs(min)) * Byte.MaxValue).toByte
+  }
+
+  def dequantize(byte: Byte, max: Float, min: Float): Float = {
```

Comment:
why do we need dequantize?

Reply:
For testing. To make sure quantizing correctly.